test/cli/test_shell.py

```diff
@@ -45,9 +42,11 @@ def fake_alternative_interpreters():
         yield
 
 
-def test_basic(pystartup):
+def test_basic(pystartup, rabbit_config):
```

Comment:
The `n` global creates a `ReplyListener` that connects to rabbit. These tests also used to assume there was a broker running at `amqp://localhost`